extensions/core/interfaces/code/interface.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div class="interface-code">
+    <codemirror ref="cm"
+                :value="options.placeholder"
+                :options="cmOptions"
+                @ready="onCmReady"
+                @focus="onCmFocus"
+                @input="onCmCodeChange"
+    ></codemirror>
+    <div class="clipboard-btn">
+      <button class="btn" @click="$emit('input', code)">
+        <i class="material-icons">assignment_turned_in</i>
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+
+// require component
```

Comment:
I'm always in favor of descriptive comments explaining the _why_, but comments saying "require css" followed by an `import x from "y.css"` aren't really adding any value, but do add visual clutter